### PR TITLE
APPS-2313: CDSS WOA-5 Remediation of ADA compliance violations

### PR DESCRIPTION
**Pull request by bergieboy**
### APPS-2314 & APPS-2326: Ensure containing elements that allow text resize without loss of functionality.
  This was raised by CDSS and the Lighthouse report in Chrome dev tools. The description and justification are below.

  - `<meta>` tags always go inside the <head> element and are typically used to specify a character set, page description, keywords, author of the document, and viewport settings. Metadata will not be displayed on the page but is machine parsable.
  - The `user-scalable` attribute defines whether users can scale (zoom) the page content on mobile. By default, scaling is on, but if user-scalable is set to '0' or 'no' then this will prevent users from zooming, which negatively affects user experience (in most cases). For accessibility reasons though, `user-scalable` needs to be set to yes and the `maximum-scale` must be at least 5.

  Current:
  `<meta name="viewport" content="width=device-width, height=device-height, initial-scale=1.0, user-scalable=no, minimum-scale=1.0, maximum-scale=2.0">`

  Should be:
  `<meta name="viewport" content="width=device-width, height=device-height, initial-scale=1.0, user-scalable=yes, minimum-scale=1.0, maximum-scale=5.0">`

### APPS-2315: Provide a valid label for form fields.
  Accessible Rich Internet Applications (ARIA) is a set of attributes that define ways to make web content and web applications (especially those developed with JavaScript) more accessible to people with disabilities.

  `aria-label` is a field that can be passed directly to `react-select`, which is below the hood of LUK's `Select`. Passing this label resolves this issue. 

### APPS-2327: Provide Adequate Contrast between placeholder and background on Select Component.
  Raised by the Lighthouse report from Chrome dev tools.

  Passing the 'themes' prop to LUK's `Select` component allows you to override the color that is used for the placeholder. I wish there were a cleaner way to achieve this, but this is the best solution I found. Open to other suggestions. 

  `const getPlaceholderTheme = (theme) => ({ ...theme, colors: { ...theme.colors, neutral50: '#555E6F' } });`

  `<Select  ... theme={getPlaceholderTheme} ... />`

### APPS-2316: Ensure ARIA roles, states, and properties are valid. [UPDATED]

  using `useRef` to set `ref` on the Wrapper component, I was able to search its children for the `input` element and set the attribute, `aria-autocomplete`, to 'none'. This should solve this issue. 

  ~As far as I can tell, this is not achievable when using `react-select`, which is what's under the hood for LUK's `Select` and only allows for `aria-label` and `aria-labeledby` to be passed directly. `aria-autocomplete` is set to list, but cannot be modified in any way that I have found yet. The PR listed below is from about a month ago in the `react-select` repo, and it would allow for you to pass `aria` properties, but it doesn't seem to have gotten much attention.~

~[PR for allowing any aria property](https://github.com/JedWatson/react-select/pull/4132)~

### CODE REVIEW

File: src/containers/location/providers/LocationSearchBar.js [hunk 1]
```diff
@@ -35,6 +35,15 @@ import { STATE, PROVIDERS } from '../../../utils/constants/StateConstants';
 
 const { media } = StyleUtils;
 
+/* placeholder color needs to be darker to provide more contrast between text and background */
+const getPlaceholderTheme = (theme) => ({
+  ...theme,
+  colors: {
+    ...theme.colors,
+    neutral50: '#555E6F'
```
Comment (UnsungHero97) [inline src/containers/location/providers/LocationSearchBar.js:43]: Please use the new LUK colors. The closest to `#555E6F ` is `NEUTRAL.N700`.

File: src/containers/location/providers/LocationSearchBar.js [hunk 2]
```diff
@@ -35,6 +35,15 @@ import { STATE, PROVIDERS } from '../../../utils/constants/StateConstants';
 
 const { media } = StyleUtils;
 
+/* placeholder color needs to be darker to provide more contrast between text and background */
+const getPlaceholderTheme = (theme) => ({
```
Comment (UnsungHero97) [inline src/containers/location/providers/LocationSearchBar.js:39]: I think this should be just `getTheme`. Also, some form of this change should also make its way into LUK.

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]

File: src/containers/location/providers/LocationSearchBar.js [hunk 3]
```diff
@@ -33,14 +34,15 @@ import { isNonEmptyString } from '../../../utils/LangUtils';
 import { LABELS } from '../../../utils/constants/Labels';
 import { STATE, PROVIDERS } from '../../../utils/constants/StateConstants';
 
+const { NEUTRALS } = Colors;
```
Comment (UnsungHero97) [inline src/containers/location/providers/LocationSearchBar.js:37]: `NEUTRALS` is the old one. `NEUTRAL` (singular) is the new one.

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]

**Review by UnsungHero97 — DISMISSED** [review]

**Review by anzioka — DISMISSED** [review]

**Review by anzioka — APPROVED** [review]

**Review by solomontang — APPROVED** [review]
Regarding the contrast of the placeholder in select, this is probably a change we wanna make to the luk repo

File: src/containers/app/AppContainer.js [hunk 4]
```diff
@@ -12,6 +12,7 @@ import {
   Spinner,
   ThemeProvider,
   lightTheme,
+  StylesProvider
```
Comment (UnsungHero97) [inline src/containers/app/AppContainer.js:15]: Please sort.

**Review by UnsungHero97 — APPROVED** [review]

**Pull request merged**